.github/ISSUE_TEMPLATE/001_bug_report.md

```diff
@@ -27,31 +29,44 @@ the problem you're facing:
  - Is this a regression? If yes, have you been able to "git bisect" it to a
    specific commit?
  - ...
+-->
 
 **To Reproduce**
+<!--
 Steps to reproduce the behavior:
 1. mkdir build; cd build
 2. cmake -DBOARD=board\_xyz
 3. make
 4. See error
+-->
 
 **Expected behavior**
+<!--
 A clear and concise description of what you expected to happen.
+-->
 
 **Impact**
+<!--
 What impact does this issue have on your progress (e.g., annoyance, showstopper)
+-->
 
 **Logs and console output**
+<!--
 If applicable, add console logs or other types of debug information
 e.g Wireshark capture or Logic analyzer capture (upload in zip archive).
 copy-and-paste text and put a code fence (\`\`\`) before and after, to help
 explain the issue. (if unable to obtain text log, add a screenshot)
+-->
 
 **Environment (please complete the following information):**
+<!--
  - OS: (e.g. Linux, MacOS, Windows)
  - Toolchain (e.g Zephyr SDK, ...)
  - Commit SHA or Version used
+-->
```

Comment:
Shouldn't this be left as-is for people to fill in?

Reply:
yeah, left as is